### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**
Updates based on https://github.com/mattermost/mattermost/blob/release-9.0/webapp/channels/package.json.

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/NOTICE.txt [hunk 1]
```diff
@@ -1420,76 +1446,6 @@ SOFTWARE.
 
 ---
 
-## emoji-datasource
```
Comment (M-ZubairAhmed) [inline webapp/channels/NOTICE.txt:1423]: What tools are you using to extract this information because this library is still being used.

Comment (amyblais) [reply webapp/channels/NOTICE.txt:1423]: I check the libraries listed in this file https://github.com/mattermost/mattermost/blob/release-9.0/webapp/channels/package.json. 

I'll add it back now.

File: webapp/channels/NOTICE.txt [hunk 2]
```diff
@@ -1455,41 +1481,6 @@ SOFTWARE.
 
 ---
 
-## emoji-datasource-apple
```
Comment (M-ZubairAhmed) [inline webapp/channels/NOTICE.txt:1458]: we use this as well https://github.com/mattermost/mattermost/blob/release-9.0/webapp/channels/package.json#L145
:D

Comment (amyblais) [reply webapp/channels/NOTICE.txt]: Added it back :)

**Review by M-ZubairAhmed — APPROVED** [review]
🎯

**Review by hmhealey — APPROVED** [review]

**Pull request merged**